### DISCUSSION TITLE
Generating charts on the server / in AWS Lambda?

### DISCUSSION THREAD

**Discussion by rpivovar-godaddy** [observablehq/plot#853] [Q&A] [answered]
I'm trying to generate a chart inside a Lambda function and I'm wondering if there is any documentation or support for doing something like this in a non-web context? I'm trying to do this with jsdom at the moment.

**Comment by mbostock** [comment, marked as answer]
Did you see https://github.com/observablehq/plot/discussions/847?

**Comment by rpivovar-godaddy** [comment]
thanks!
